### ISSUE TITLE
Don't commit .askjson and .askcode anymore

### ISSUE THREAD

**Issue by czerwinskilukasz1** [xFAANG/askql#170]
Following a discussion on [Discord channel](https://discord.com/channels/688128418914500838/704117328735895662/722427091210993714), eventually, once we get to certain level of stability, we could replace .askjson and .askcode with some unit tests and stop commit these files.

This is a task for the future, several things need to happen before we consider doing it.

**Comment by YonatanKra** [comment]
I do not believe these files need to be committed - they can be just placed in `.gitignore`. They are being auto-generated on every test run anyway...
What do you think? Would that be a good solution?

**Comment by mhagmajer** [comment]
@YonatanKra @czerwinskilukasz1 let's come up with a long .ask file that uses all of our constructs and lets commit snapshots only for this file. This way we keep most value of snapshots while decluttering PRs. Who's up for it?

**Comment by czerwinskilukasz1** [comment]
Tasks:

- [ ] Create one example per AskScript construct in `src/askscript/__tests__/00-documentation-examples/` and add them to Documentation on Notion
- [ ] Stop creating .snapshot.jsx and .askcode for tests in other folders (configured in a config file)
